### PR TITLE
New PHP-Client with Go Local Service

### PR DESCRIPTION
**Pull request by vmsachin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,69 +7,190 @@ An [Aerospike](https://www.aerospike.com/) client library for PHP 8+.
...
+* Rustc v1.74 =<
+* Go Toolchain [Go Toolchains - The Go Programming Language](https://go.dev/doc/toolchain)
+* Protobuf Compiler [protoc-gen-go command - google.golang.org/protobuf/cmd/protoc-gen-go - Go Packages](https://pkg.go.dev/google.golang.org/protobuf/cmd/protoc-gen-go)
+* ext-php-rs v0.12.0 [github repository link](https://github.com/davidcole1340/ext-php-rs/tree/master)
 
 ## Current Limitations
 
-* Does not support Scan/Query API features for Aerospike server 6.4.
+* Does not support Scan/Query API features/
 * Use Aerospike Server 6.3 for testing, the Rust client does not fully support newer versions.
```
Comment (justinlee-aerospike) [inline README.md:22]: The line "The Rust client does not fully support newer versions"  should be removed I think.  With the underlying Go client this should no longer be a limitation.

File: README.md [hunk 2]
```diff
@@ -7,69 +7,190 @@ An [Aerospike](https://www.aerospike.com/) client library for PHP 8+.
...
+* Should have the local go daemon setup.
```
Comment (justinlee-aerospike) [inline README.md:23]: Do you also need to mention that CDTs are not supported?

**Review by justinlee-aerospike — CHANGES_REQUESTED** [review]
Looks good.  Remove the line in the main README.md that says it only works for 6.3 and I think you will be good to go.

File: README.md (hunk 1 above)
Comment (vmsachin) [reply README.md:22]: yeah, I'll remove that

File: README.md (hunk 2 above)
Comment (vmsachin) [reply README.md:23]: sure, I could add that as well.

**Review by justinlee-aerospike — APPROVED** [review]
changes look good

**Pull request merged**